src/styles/layouts/_layout-about-us.scss

```diff
@@ -10,15 +10,20 @@
   height: 100vh;
 
   @include respond-above('md') {
-    grid-template-rows: map-get($header-height, 'md') 1fr map-get($footer-height, 'md');
+    grid-template-rows: map-get($header-height, 'md') 1fr map-get($footer-height,'md');
   }
 
   @include respond-above('lg') {
-    grid-template-rows: map-get($header-height, 'lg') 1fr map-get($footer-height, 'lg');
+    grid-template-rows: map-get($header-height, 'lg') 1fr map-get($footer-height,'lg');
   }
 
   @include respond-above('xl') {
-    grid-template-rows: map-get($header-height, 'xl') 1fr map-get($footer-height, 'xl');
+    grid-template-columns: 1fr minmax(rem(1104px), rem(1440px)) 1fr;
+    grid-template-rows: map-get($header-height, 'xl') 1fr map-get($footer-height,'xl');
```

Comment:
I think we should add the `column-gap` back. When the context hits the side of the browser it looks kind of 
![Screen Shot 2564-02-26 at 01 29 00](https://user-images.githubusercontent.com/32289827/109201450-708d1b80-77d4-11eb-8fe9-7978060416fb.png)
ugly.